docs/reference/developer/concepts.ipynb

```diff
@@ -38,21 +38,23 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
-    "assert 'x' in da['x', 0:1].coords # range slice preserves coord\n",
-    "assert 'x' in da['x', 0:1].masks # range slice preserves coord\n",
-    "assert 'x' in da['x', 0].attrs # point slice converts coord to attr\n",
-    "assert 'x' not in da['x', 0].coords\n",
-    "assert 'x' in da['x', 0].attrs\n",
-    "assert 'x' in da['x', 0].masks # point slice preserves masks as aligned"
+    "assert da['x', 0:1].coords['x'].aligned  # range slice preserves coord and alignment\n",
+    "assert 'x' in da['x', 0:1].masks  # range slice preserves mask\n",
+    "assert not da['x', 0].coords['x'].aligned  # point slice converts makes coord unaligned\n",
```

Comment:
```suggestion
    "assert not da['x', 0].coords['x'].aligned  # point slice makes coord unaligned\n",
```

Reply:
No it doesn't. We said that `identical` for variables shouldn't check (like readonly). And data array simply maps this check over the coords.
I'm not happy with the form of these tests. But I'm unsure if we should change the behaviour of identical (for data array) in this way.

Reply:
> No it doesn't. We said that identical for variables shouldn't check (like readonly). And data array simply maps this check over the coords.

I am not suggesting to change the behavior of `identical` for `Variable`. But for `DataArray` I think it has to, otherwise it will become quite useless?